admin/access-control/manage.md

```diff
@@ -65,6 +66,20 @@ you:
   to log in with an email/password option when you've enabled OIDC
   authentication
 
+### Logging
+
+If you're having issues with your OIDC configuration, you can enable additional
+logging of OIDC tokens to aid in troubleshooting.
+
+To do so, [update your Helm chart](../../guides/admin/helm-charts.md) and set
+the `OIDC_DEBUG` environment variable to `true`:
+
+```yaml
+coderd:
+  extraEnvs:
+    - "OIDC_DEBUG=true"
```

Comment:
Not sure which convention we prefer, but we should probably stick to one consistently:

```suggestion
    - name: "OIDC_DEBUG"
      value: "true"
```